tests/forte/data/data_store_test.py

```diff
@@ -584,6 +584,66 @@ def test_add_annotation_raw(self):
             ],
         )
 
+    def test_add_audio_annotation_raw(self):
+        # test add Document entry
+        tid_recording: int = self.data_store.add_audio_annotation_raw(
+            "ft.onto.base_ontology.Recording", 1, 5
+        )
+        # test add Sentence entry
+        tid_audio_utterance: int = self.data_store.add_audio_annotation_raw(
+            "ft.onto.base_ontology.AudioUtterance", 5, 8
+        )
+        tid_utterance: int = self.data_store.add_audio_annotation_raw(
+            "ft.onto.base_ontology.Utterance", 5, 8
```

Comment:
`ft.onto.base_ontology.Utterance` is not a subclass of `AudioAnnotation`. Not sure if it's okay to add it using `add_audio_annotation_raw`.

Reply:
Fixed it.